### PR TITLE
Fixed a bug filtering jobs by start time taking too much time.

### PR DESCRIPTION
**Pull request by zacharyhsia**

### CODE REVIEW

**Comment by zacharyhsia** [comment]
Ready for review. Thanks!

File: ui/src/app/dashboard/dashboard.resolver.service.ts [hunk 1]
```diff
@@ -11,6 +12,7 @@ export class DashboardResolver implements Resolve<AggregationResponse>{
   ) {}
 
   resolve(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Promise<AggregationResponse> {
-    return this.jobManagerService.queryAggregations(TimeFrame.DAYS7, route.queryParams['projectId']);
+    return this.jobManagerService.queryAggregations(TimeFrame.DAYS7,
```
Comment (calbach) [inline ui/src/app/dashboard/dashboard.resolver.service.ts]: This change seems unrelated to this PR.

File: servers/dsub/jobs/controllers/jobs_controller.py [hunk 2]
```diff
@@ -22,6 +22,7 @@
 
 _DEFAULT_PAGE_SIZE = 32
 _MAX_PAGE_SIZE = 256
+_MAX_PENDING_TIME_IN_DAY = 7
```
Comment (calbach) [inline servers/dsub/jobs/controllers/jobs_controller.py]: nit: I would replace "IN_DAY" with "DAYS".

File: servers/dsub/jobs/controllers/jobs_controller.py [hunk 3]
```diff
@@ -114,6 +115,53 @@ def query_jobs(body):
     Returns:
         QueryJobsResponse: Response containing results from query
     """
+    job_generator = _get_job_generator(body)
```
Comment (calbach) [inline servers/dsub/jobs/controllers/jobs_controller.py]: What are all these other changes? Are these related to the PR?
